apps/server/src/modules/learnroom/controller/dto/dashboard.response.ts

```diff
@@ -1,29 +1,83 @@
 import { ApiProperty } from '@nestjs/swagger';
+import { LearnroomMetadata, GridElementWithPosition, DashboardEntity } from '@shared/domain';
 
 export class DashboardGridSubElementResponse {
+	private constructor({ id, title, shortTitle, displayColor }: DashboardGridSubElementResponse) {
+		this.id = id;
+		this.title = title;
+		this.shortTitle = shortTitle;
+		this.displayColor = displayColor;
+	}
+
+	static FromLearnroomMetadata(metadata: LearnroomMetadata): DashboardGridSubElementResponse {
```

Comment:
why do you here need to know about lernroom metadata? 
isn't this just defining how the api looks like? the dto is only a container for data structures.
suggestion: use a separate mapper, which is used in a specific case (in the controller [where we do it]/usecase[where it would happen for strict ddd]) where the knowledge about what is lernroom metadata exists.

Reply:
yes, this has to be moved to mappers. also the DTOs sshould just have simple public constructors. they're just a place to define the type and structure of our api payloads.